recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        return self.soup.find("ul", {"class": "recipe-specs"}).find_all("li")
+
+    def total_time(self):
+        time = self._specs()[0].find_next("span").get_text()
+        return get_minutes(re.sub(r" \S$| ", "", time))
+
+    def category(self):
+        return normalize_string(self._specs()[1].find_next("span").get_text())
+
+    def nutrients(self):
+        nutrients = self._specs()[2].find_next("p").get_text()
+        nutrients = nutrients.split(",")
+        nutrients = [nutrient.split()[1] for nutrient in nutrients]
+        nutrients = {
+            "calories": nutrients[0] + " calories",
+            "fatContent": nutrients[1] + " grams fat",
+            "proteinContent": nutrients[2] + " grams protein",
+            "carbohydrateContent": nutrients[3] + " grams carbohydrates",
+        }
+        return nutrients
```

Comment:
Thanks for the pull request @tomtjes!

Regarding the nutrient info retrieval here: I'm worried that the order of the nutrients in the parsed element text (the elements containing the text `Kalorien ...` and so on) might differ between recipe webpages -- or perhaps change in future -- in ways that could cause this to return incorrect results.

For me to approve this pull request, I'd want to see some safety guards put in place to ensure that if we return something indicating `calories` in the output value, that the text "calories" (or internationalized equivalent for the `language` of the page) was found alongside the quantity.

Reply:
Ah, I first had a solution where I built a vocabulary and looked for the words depending on the language tag - until I realized that all recipes that I looked at have the same four nutrients in the same order. But I can see if I can recreate the keyword-based solution.

Reply:
Ok, thank you - that's reassuring that it is the same for multiple pages; even so I think we should be cautious about the possibility that the `recipe-spec` items might be re-ordered (or additional ones inserted between the existing entries), and similarly with the nutrient listings.